index.html

```diff
@@ -21,17 +21,40 @@ <h1 class="mh-logo">
                 
             </h1>
     
-              <h1 class="button">
-                  <a href="#"><img alt="hambutton" src="./images/hambutton.svg"></a>
-              </h1>
+             
+              <div class="desktop-buttons">
+                <ul class="main-nav-list">
+                  <li>
+                      <a href="#">Portfolio</a>
+                  </li>
+                  <li>
+                      <a href="#">About</a>
+                  </li>
+                  <li>
+                      <a href="#">Contact</a>
+                  </li>
+                  <li>
+                      <a href="#"><img src="images/mail.svg" alt=""></a>
+                  </li>
+              </ul>
+              </div>
+              <div class="ham-button">
+                <a href="#"><img alt="hambutton" src="./images/hambutton.svg"></a>
+            </div>
         </div>
 
     </header>
 
 
     <section class="body-container">
-   
 
+   
+      <section class="intro-container">
+        <ul class="socials">
+          <li><a href='https://github.com/FabricioLatl/'><img alt="github" src='images/github.svg' /></a></li>
+          <li><a href='https://www.linkedin.com/in/fabriciolatl/'><img alt="linkedin" src='images/linkedin.svg' /></a></li> 
+          <li><a href='https://twitter.com/Wafer_lt'><img alt="twitter" src='images/twitter.svg' /></a></li>
+        </ul>
       <ul class="intro">
         <li>
           <h1 class="name">
```

Comment:
- [ ] I think it will be good if you remove the extra space at the right of the whole page to match the Figma design. You can try to hide the `overflow-x` of your document(i.e.: `html { overflow-x: hidden; }`) I think this can try to fix the issue. But you should check the `witdh` of your inputs' form as you gave them fixed width.
![Screenshot from 2022-07-16 16-46-24](https://user-images.githubusercontent.com/83042742/179359874-fdbdeec6-1bb6-44ad-8d2e-ab82076a252d.png)